internal/storage/store_test.go

```diff
@@ -13,50 +12,50 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-// TODO turn this into a general in memory store service (HF)
-type mockStore struct {
-	buffer bytes.Buffer
-}
+// // TODO turn this into a general in memory store service (HF)
```

Comment:
@kishansagathiya Did you forget to change this part?

Reply:
@kishansagathiya are we dropping the implementation?

Reply:
It is moved to mock_store.go and exported